encord/client_metadata_schema.py

```diff
@@ -1,26 +1,83 @@
-from typing import Dict, Optional
+import functools
+from typing import Any, Dict, Optional
 
-import encord.orm.client_metadata_schema as orm
+from encord.exceptions import MetadataValidationError
 from encord.http.v2.api_client import ApiClient
+from encord.orm.client_metadata_schema import (
+    ClientMetadataSchema,
+    ClientMetadataSchemaPayload,
+    ClientMetadataSchemaTypes,
+    validate_embedding,
+)
 
 
-def set_client_metadata_schema_from_dict(api_client: ApiClient, json_dict: Dict[str, orm.ClientMetadataSchemaTypes]):
+@functools.lru_cache(maxsize=128)
```

Comment:
Idea being we need a reference somehow to client_metadata_schema and my original approaches were really struggling with Import errors. This isn't fool proof and I don't love it.